CHANGELOG.md

```diff
@@ -8,7 +8,8 @@
 
 ### Chores
 
-- Fixed locale in Storybook ([PR 3062](https://github.com/input-output-hk/daedalus/pull/3062))
+- Fixed Language components in Storybook to have it's own mocked store ([PR 3063](https://github.com/input-output-hk/daedalus/pull/3063))
+- Fixed locale in Storybook ([PR 3042](https://github.com/input-output-hk/daedalus/pull/3042))
```

Comment:
```suggestion
- Fixed locale in Storybook ([PR 3062](https://github.com/input-output-hk/daedalus/pull/3062))
```

Reply:
@szymonmaslowski @VanessaPC We can retest this if required. No problem 😄